distribution/openrct2.d.ts

```diff
@@ -2606,6 +2606,56 @@ declare global {
         "waiting_to_depart" |
         "waiting_to_start";
 
+    type CrashParticleType =  0 | 1 | 2 | 3 | 4;
+
+    /**
+     * Represents a vehicle explosion particle. They are emitted during a vehicle
+     * crash and will bounce until their timer expires and they are automatically
+     * removed.
+     */
+    interface CrashedVehicleParticle extends Entity {
+        /**
+         * The colour of the particle. Only body and trim are used.
+         */
+        colours: VehicleColour;
+
+        /**
+         * The lifetime of the particle in ticks. Default value 65535. Entity is
+         * automatically removed at 0.
+         */
+        timeToLive: number;
+
+        /**
+         * The particle velocity.
+         */
+        velocity: CoordsXYZ;
+
+        /**
+         * The acceleration of the particle in the x, y, and z directions.
+         */
+        acceleration: CoordsXYZ;
+
+        /**
+         * The type of crash particle.
+         */
+        crashParticleType: CrashParticleType;
+
+        /**
+         * The current frame of the crash particle.
+         */
+        frame: number;
+
+        /**
+         * Sets a random type, random frame, random lifespan, and random initial velocity.
+         */
+        launchRandom(colours: VehicleColour): void;
+
+        /**
+         * Sets a random type, random frame, random lifespan.
+         */
+        launch(colours: VehicleColour, initialVelocity: CoordsXYZ): void;
```

Comment:
Kind of a nitpick, but wouldn't it be better to just have one method for this? 😅 The second method still has random elements despite its name not mentioning it. And isn't the second one basically the same as this?
```js
particle.launchRandom(colours);
particle.acceleration = initialVelocity; // Also interesting confusion in naming here.
```